src/haddock/gear/prepare_run.py

```diff
@@ -219,9 +220,16 @@ def validate_modules_params(modules_params):
             - expandable_params
 
         if diff:
+            matched = fuzzy_match(diff, defaults.keys())
+
+            def pretty_print(match):
+                return f" * \'{match[0]}\' did you mean \'{match[1]}\'?"
+
+            eol = '\n'
```

Comment:
in python:
```python
import os
os.linesep.join(...)
```
could you change? This get's you OS-specific line break character.